docs/OESDLBugBar.md

```diff
@@ -0,0 +1,51 @@
+# Open Enclave SDL Bug Bar
+
+The information listed in this “bug bar” is intended to help Open Enclave SDK developers to triage bugs and determine the bug severity in terms of security for all types of Trusted Execution Environment (TEE) supported by Open Enclave. It is highly recommended to fix all known bugs with critical, important, or moderate severity prior to release. This bug bar should also be used during the triage meeting to evaluate bugs for any security implication and tag accordingly.
+
+The bug bar describes different severities for the bugs affecting TEE. TEEs are specifically designed to provide: Confidentiality, Integrity, Authenticity and, to a more limited extent, Authorization. There is a security boundary between TEE and rest of the system, which needs to be defended against any bugs that lead to compromising the TEE purpose. Anything within TEE should never be accessible or modified from outside.
+
+## Open Enclave – Severity Type Pivot
+-------------------------------------
+
+* ### **Critical**
+  Information Disclosure
+  * Any cases where an attacker can bypass the TEE security boundary to read arbitrary information belonging to a TEE that was not intended or designed to be exposed
```

Comment:
"read" has a fairly specific connotation. Perhaps change to reveal.

"belonging to a TEE" --> within a TEE